src/scores/pandas/continuous.py

```diff
@@ -0,0 +1,98 @@
+        angular: specifies whether `fcst` and `obs` are angular
+            data (e.g. wind direction). If True, a different function is used
+            to calculate the difference between `fcst` and `obs`, which
+            accounts for circularity. Angular `fcst` and `obs` data should be in
+            degrees rather than radians.
+
+    Returns:
+        An object containing
+            a single floating point number representing the mean squared
+            error for the supplied data. All dimensions will be reduced.
+
+    """
+    return __continuous.mse(fcst, obs, angular=angular)
+
+
+def rmse(
+    fcst: PandasType,
+    obs: PandasType,
+    angular: bool = False,
+) -> PandasType:
+    """Calculate the Root Mean Squared Error from xarray or pandas objects.
+
+    A detailed explanation is on [Wikipedia](https://en.wikipedia.org/wiki/Root-mean-square_deviation)
+
+    Dimensional reduction is not supported for pandas and the user should
+    convert their data to xarray to formulate the call to the base metric, `scores.continuous.rmse`.
+
+    Args:
+        fcst: Forecast or predicted variables in pandas.
+        obs: Observed variables in pandas.
+        angular: specifies whether `fcst` and `obs` are angular
+            data (e.g. wind direction). If True, a different function is used
+            to calculate the difference between `fcst` and `obs`, which
+            accounts for circularity. Angular `fcst` and `obs` data should be in
+            degrees rather than radians.
+
+    Returns:
+        An object containing
+            a single floating point number representing the root mean squared
+            error for the supplied data. All dimensions will be reduced.
+
+    """
+    return __continuous.rmse(fcst, obs, angular=angular)
+
+
+def mae(
+    fcst: PandasType,
+    obs: PandasType,
+    angular: bool = False,
+) -> PandasType:
+    """Calculates the mean absolute error from forecast and observed data.
+
+    A detailed explanation is on [Wikipedia](https://en.wikipedia.org/wiki/Mean_absolute_error)
+
+
+    Dimensional reduction is not supported for pandas and the user should
+    convert their data to xarray to formulate the call to the base metric, `scores.continuous.mae`.
+
+    Args:
+        fcst: Forecast or predicted variables in pandas.
+        obs: Observed variables in pandas.
+        angular: specifies whether `fcst` and `obs` are angular
+            data (e.g. wind direction). If True, a different function is used
+            to calculate the difference between `fcst` and `obs`, which
+            accounts for circularity. Angular `fcst` and `obs` data should be in
+            degrees rather than radians.
+
+    Returns:
+        An object containing
+            a single floating point number representing the mean absolute
+            error for the supplied data. All dimensions will be reduced.
+
+    """
+    return __continuous.mae(fcst, obs, angular=angular)
```

Comment:
It would be good to add another metric that is slightly more complicated than these initial metrics (e.g., FIRM, or quantile loss) to see additional work needs to be done to get it to work with Pandas. This could be done in this pull request or different one, but should be done soon.

Reply:
After some discussion, my plan is to treat the pandas API as unstable for right now to enable progress to be made and gain more experience through utilising the API. We should aim to stabilise this by version 1, but it will also mean that we can be more atomic with the merge requests and perform smaller iterations. I agree that adding FIRM and exploring more complex metrics is important, but it is also valuable to start making the API available for people to test through use.